overlord/snapstate/snapmgr_test.go

```diff
@@ -320,6 +314,30 @@ func (s *snapmgrTestSuite) TestUpdateTasks(c *C) {
 	c.Check(ss.Channel, Equals, "some-channel")
 }
 
+func (s *snapmgrTestSuite) TestUpdateTasksToLocalRevision(c *C) {
```

Comment:
wondering if it be should be called (for consistency) TestUpdateToLocalRevisionTasks ?